test/TestImportTasks.py

```diff
@@ -0,0 +1,73 @@
+"""Calling 'import_task' from multiple files in different directories."""
+import os
+import shutil
+import tempfile
+import textwrap
+
+import pytest
+
+from ansiblelint.runner import Runner
+
+LAYOUT = {
+    'directories': ['tasks', 'tasks/subtasks'],
+    'files': {
+        'main.yml': textwrap.dedent(
+            """\
+            ---
+            - hosts: target
+              gather_facts: false
+              tasks:
+                - name: from main import task 1
+                  import_tasks: tasks/task_1.yml
```

Comment:
Since you mentioned in the issue that `include_tasks` is also affected, it'd be reasonable to use `@pytest.mark.parametrize` on the test to verify both scenarios.